app/controllers/v0/health_records_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+module V0
+  class HealthRecordsController < BBController
+    def refresh
+      resource = client.get_extract_status
```

Comment:
I see in `BB::Client`, get_extract_status "should be called once per user, will take up to 15 minutes" - does that mean if this API is invoked multiple times for a user it passes through multiple calls to MHV, and is that problematic, or is it idempotent on their end?